README.md

```diff
@@ -40,10 +40,20 @@ Alternatively, dependencies can be installed into a [Python Virtual Environment]
     $ wget -nd http://www.tmbdev.net/en-default.pyrnn.gz
     $ mv en-default.pyrnn.gz models/
 
+An additional method using [Conda](http://conda.pydata.org/) is also possible:
+
+    $ conda create -n ocropus_env python=2.7
+    $ source activate ocropus_env
+    # PyPI and conda disagree on the name of the PyTables package:
```

Comment:
The comment is obsolete now I think, though it is still true of course ;)

And can you use a fenced code block and skip the `$`? Then users can copy and paste the whole thing.

```sh
conda create -n ocropus_env python-2.7
source activate ocropus_env
...
```